src/code/qualification/plotQualificationTimeProfile.m

```diff
@@ -85,6 +85,14 @@ function plotQualificationTimeProfile(WSettings,plotIndex,TimeProfile,ObservedDa
 end
 legend(curvesHandle, curvesLegend);
 
+% Scale X-Axis according to data if xlim is not provided
+if ~isfield(xAxesOptions, 'Min') && ~isfield(xAxesOptions, 'Max')
+    PlottedData = get(gca, 'children');
+    Xmin = min(cellfun(@min, arrayfun(@(x) getfield(x, 'XData'), PlottedData, 'UniformOutput', false)));
+    Xmax = max(cellfun(@max, arrayfun(@(x) getfield(x, 'XData'), PlottedData, 'UniformOutput', false)));
+    xlim([Xmin Xmax]);
+end
+

```

Comment:
@pchelle shouldn't it be:

```
if ~isfield(xAxesOptions, 'Min')
  Xmin = ...
end
if ~isfield(xAxesOptions, 'Max')
  Xmax = ...
end
```

instead of
```
if ~isfield(xAxesOptions, 'Min') && ~isfield(xAxesOptions, 'Max')
  Xmin=...
  Xmax=...
end
```
?

Reply:
Okay. I was not sure that the user could provide only one of them.